frameworks/g-cloud-12/questions/services/webInterfaceUsage.yml

```diff
@@ -18,8 +18,8 @@ max_length_in_words: 200
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter description.
   - name: under_200_words
-    message: Your description must be no more than 200 words.
+    message: Description must be 200 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?